tests/e2e/specs/api-cache.test.js

```diff
@@ -28,15 +28,29 @@ import {
 	resetSiteKit,
 	safeLoginUser,
 	setupSiteKit,
-	testSiteNotification,
 	useRequestInterception,
-	wpApiFetch,
 } from '../utils';
 import { deleteAuthCookie } from '../utils/delete-auth-cookie';
 
-const goToSiteKitDashboard = async () => {
-	await visitAdminPage( 'admin.php', 'page=googlesitekit-dashboard' );
-};
+async function goToWordPressDashboard() {
+	await visitAdminPage( 'index.php' );
+}
+
+async function googleSiteKitAPIgetTime( options ) {
```

Comment:
The case for this function name needs a tweak:

```suggestion
async function googleSiteKitAPIGetTime( options ) {
```